stress_test.py

```diff
@@ -69,32 +70,47 @@ def delete(self):
 
 class StressTestRunner:
     """
-    Abstract class that holds common logic for stress test runners.
+    This runner object holds the logic on how the stress tests are run based on the command line
+    arguments passed in.
 
     Args:
         cl: cl instance used to run CodaLab commands
         args: command line arguments
     """
 
-    # List of random, large docker images
+    # List of random, large Docker images in alphabetical order
```

Comment:
Why specify "in alphabetical order"?

Reply:
This is just for better organization. It's easy to which images are already in the list.